src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -16,13 +16,13 @@ class ReportsBox extends React.Component {
       dateTo: new Date(),
     };
 
-    this.getVisitsByPeriod = this.getVisitsByPeriod.bind(this);
+    this.visitsByPeriod = this.visitsByPeriod.bind(this);
   }
 
-  async getVisitsByPeriod(domainName, periodSelectedDays) {
+  async visitsByPeriod(domainName, periodSelectedDays) {
```

Comment:
What do you think about calling it _update_, _load_ or _fetch_?
```suggestion
  async fetchVisitsByPeriod(domainName, periodSelectedDays) {
```